R/arabic2kansuji.R

```diff
@@ -56,7 +56,7 @@ arabic2kansuji <- function(str,
 #' @importFrom stats na.omit
 #'
 arabic2kansuji_cal <- function(num, ...){
-  if(length(num) > 1) stop( writeLines("only one number can convert to kansuji. \nuse `arabic2kansuji_cal` to convert to over 2 numbers."))
+  if(length(num) > 1) stop("only one number can convert to kansuji. use `arabic2kansuji_cal` to convert to over 2 numbers.")
```

Comment:
The error message was on multiple lines, but the line feeds look strange depending on the environment, so to leave it to the line feeds of each environment.